src/integrationTest/java/uk/gov/hmcts/reform/iacaseapi/component/testutils/FeeRegisterIntegrationTest.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.iacaseapi.component.testutils;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static org.assertj.core.api.Assertions.*;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import org.junit.Before;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.Resource;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.fee.FeeResponse;
+
+public abstract class FeeRegisterIntegrationTest extends SpringBootIntegrationTest {
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @org.springframework.beans.factory.annotation.Value("classpath:fees-register-api-response.json")
+    private Resource resourceFile;
+
+    private String feeRegisterUri
+            = "/fees-register/fees/lookup?channel=default&event=issue&jurisdiction1=tribunal&jurisdiction2=immigration%20and%20asylum%20chamber&keyword=ABC&service=other";
+
+    protected FeeResponse feeResponse;
+
+    @Before
+    public void setUpFeeRegisterStub() throws IOException {
+
+        String feeResponseJson =
+            new String(Files.readAllBytes(Paths.get(resourceFile.getURI())));
+
+        assertThat(feeResponseJson).isNotNull();
+
+        feeResponse = objectMapper.readValue(feeResponseJson, FeeResponse.class);
+
+        stubFor(get(urlEqualTo(feeRegisterUri))
```

Comment:
Maybe this setup stuff should just go into `SpringBootIntegrationTest`?  We already stub s2s and idam there.  Can put another @Before method there?